### PR TITLE
No-Handed humans can no longer strip people

### LINKED ISSUE #12408: Items can be removed by people with no hands.
Yes. Exactly this.

### PR DESCRIPTION
**Pull request by Shadow-Quill**
As per title.
tldr; humans can no longer interact with the strip panel if they don't have the hands to do so.
Fixes #12408

### CODE REVIEW

**Comment by Shadow-Quill** [comment]
!review

File: code/modules/mob/living/carbon/human/stripping.dm [hunk 1]
```diff
@@ -6,6 +6,15 @@
 		user << browse(null, text("window=mob[src.name]"))
 		return FALSE
 
+	if(ishuman(user))
+		// If you're a human and don't have hands, you shouldn't be able to strip someone.
+		var/mob/living/carbon/human/H = user
+		var/obj/item/organ/external/r_hand = H.organs_by_name[BP_L_HAND]
+		var/obj/item/organ/external/l_hand = H.organs_by_name[BP_R_HAND]
```
Comment (Geevies) [inline code/modules/mob/living/carbon/human/stripping.dm:13]: should use get_organ(BP_BLAH) here for both

**Review by Geevies — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/human/stripping.dm (hunk 1 above)
Comment (Shadow-Quill) [reply code/modules/mob/living/carbon/human/stripping.dm:13]: Code changes from earlier trying to make it work. Will do soon.

**Review by Geevies — APPROVED** [review]

**Comment by Geevies** [comment]
honestly might be worth putting that in as a generic helper, but it's fine either way

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**